### ISSUE TITLE
How to use miners with 'MultipleLosses'?

### ISSUE THREAD

**Issue by hamidreza86** [KevinMusgrave/pytorch-metric-learning#212]
Regarding multiple losses, only two dictionary are given as input arguments, the first is 'losses' and the latter is 'weights', for example see the following codes:

loss1 = losses.losses.MultiSimilarityLoss()
loss2 = losses.TupletMarginLoss()
complete_loss = losses.MultipleLosses([loss1, loss2], weights=[w1, w2])

**Now, how can we import _miners_ to' MultipleLosses'?** 
for example, if we want to use MultiSimilarity Miner for computing loss1?

**Comment by KevinMusgrave** [comment]
Thanks for bringing this up. You can do this now in the pre-release version:
```python
pip install pytorch-metric-learning==0.9.94.dev0
```

Here's an example:
```python
## Use a miner on loss 2 ##
## in list form ##
loss1 = losses.MultiSimilarityLoss()
loss2 = losses.TupletMarginLoss()
miner = miners.MultiSimilarityMiner()
complete_loss = losses.MultipleLosses([loss1, loss2], weights=[w1, w2], miners=[None, miner])

## or in dictionary form ##
complete_loss = losses.MultipleLosses({"loss1": loss1,  "loss2": loss2}, weights={"loss1": w1, "loss2": w2}, miners={"loss2": miner})
```
